CTLD.lua

```diff
@@ -5224,11 +5224,11 @@ function ctld.addF10MenuOptions()
 
                             if ctld.unitCanCarryVehicles(_unit) == false then
 
-                                -- local _cratePath = missionCommands.addSubMenuForGroup(_groupId, "Spawn Crate", _rootPath)
+                                local _cratePath = missionCommands.addSubMenuForGroup(_groupId, "Vehicle / FOB Crates", _rootPath)
                                 -- add menu for spawning crates
                                 for _subMenuName, _crates in pairs(ctld.spawnableCrates) do
 
-                                    local _cratePath = missionCommands.addSubMenuForGroup(_groupId, _subMenuName, _rootPath)
+                                    local _cratePath = missionCommands.addSubMenuForGroup(_groupId, _subMenuName, _cratePath)
```

Comment:
I'm not convinced on the utility of this change.  You have to go through an extra menu and 4 F-keys to spawn a crate instead of 3.  This also breaks saved macros and VoiceAttack profiles.

Reply:
This feature was requested multiple times; I decided to include it because it meant that we could add more crate types to the menu. I'll add pagination later

Reply:
It makes more sense now with more categories and items added to the lists.